.drone.yml

```diff
@@ -0,0 +1,68 @@
+---
+workspace:
+  base: /go
+  path: src/github.com/NYTimes/drone-gke
+
+build_config: &config
+  image: golang:1.8
+  environment:
+    - CGO_ENABLED=0
+  commands:
+    - go build -a -ldflags "-X main.rev=${DRONE_COMMIT}"
+
+pipeline:
+  test:
+    image: golang:1.8
+    environment:
+      - CGO_ENABLED=0
+    commands:
+      - go vet
+      - go test -cover -coverprofile=coverage.out
+
+  build:
+    <<: *config
+    when:
+      branch:
+        - develop
+        - master
+
+  build_release:
```

Comment:
What's the difference between `build` and `build_release` besides the conditions? Can they be merged into one?

Reply:
The only difference is the conditions. AFAIK they cannot be merged because of that.